ruby/ql/lib/codeql/ruby/frameworks/http_clients/NetHttp.qll

```diff
@@ -12,15 +12,22 @@ private import codeql.ruby.DataFlow
 /**
  * A `Net::HTTP` call which initiates an HTTP request.
  * ```ruby
+ * # one-off request
```

Comment:
This file needs to be auto-formatted, using `codeql query format --in-place NetHttp.qll`.